cmd/check_container.go

```diff
@@ -63,6 +66,23 @@ var checkContainerCmd = &cobra.Command{
 			return err
 		}
 
+		projectId := viper.GetString("pyxis-project-id")
+		if projectId == "" {
+			return errors.ErrEmptyProjectID
+		}
+		if strings.HasPrefix(projectId, "ospid-") {
+			projectId = strings.Split(projectId, "-")[1]
+		}
+		apiToken := viper.GetString("pyxis-api-token")
+		ctx := context.Background()
+		pyxisEngine := pyxis.NewPyxisEngine(apiToken, projectId, &http.Client{})
+		project, err := pyxisEngine.GetProject(ctx)
+		if err != nil {
+			log.Error(err, "could not retrieve project")
+			return err
+		}
+		log.Debugf("Project is: %s", project.Name)
```

Comment:
It would be great to be more specific of what kind of project is this. Sth like "Cert project is: " is a bit more specific.